python/raft-dask/pyproject.toml

```
@@ -35,12 +35,13 @@ license = { text = "Apache 2.0" }
 requires-python = ">=3.9"
 dependencies = [
     "dask-cuda==23.12.*",
-    "dask>=2023.7.1",
-    "distributed>=2023.7.1",
+    "dask>=2023.9.2",
+    "distributed>=2023.9.2",
```

```suggestion
    "dask==2023.9.2",
    "distributed==2023.9.2",
```

Can we leave this as hard pinnings similar to https://github.com/rapidsai/raft/pull/1864/files in `23.12` for now? I'll unpin as the release is done because we have seen inconsistent environments before if one library has flexible pins and others don't. Same in the conda files too.